Extensions/mutualchecker.js

```diff
@@ -0,0 +1,83 @@
+//* TITLE Mutual Checker **//
+//* VERSION 1.0.0 **//
+//* DESCRIPTION Adds an icon next to usernames to indicate if they follow you. **//
+//* DETAILS Does what it says on the tin! Doesn't work on sideblogs. Made from fresh, distilled Profiler juice. **//
+//* DEVELOPER AprilSylph **//
+//* FRAME false **//
+//* BETA false **//
+
+XKit.extensions.mutualchecker = new Object({
+
+	running: false,
+
+	add_icons: function() {
+		if ($(".follower").length > 0) {
+			$(".follower").each(function() {
+				var m_url = $(this).find(".name").text();
+				$(this).find(".name").append("<span class=\"mutualcheck " + m_url + "\"></span>");
+			});
+		}
+		var posts = XKit.interface.get_posts("mutualchecker-done");
+		$(posts).each(function() {
+			$(this).addClass("mutualchecker-done");
+			var m_post = XKit.interface.post($(this));
+			if (XKit.interface.where().inbox !== true) {
+				if (m_post.is_mine === true) { return; }
+			}
+			var post_owner = m_post.owner;
+			if (m_post.type === "note" && XKit.interface.where().inbox === true) {
+				var m_json_info = $(this).find(".post_avatar_link").attr('data-tumblelog-popover');
+				try {
+					var m_json_obj = JSON.parse(m_json_info);
+					post_owner = m_json_obj.name;
+				} catch (e) {
+					return;
+				}
+			}
+			var name_div_container;
+			var name_div;
+			if ($(this).find(".post_info_fence").length > 0) {
```

Comment:
this whole block can be simplified into `$(".post_info_fence,.post_info").first()`, I think?